src/error/specification

```diff
@@ -1230,3 +1230,17 @@ severity:warning
 ingroup:plugin
 module:zeromqsend
 macro:ZEROMQSEND_ERROR
+
+number: 197
+description:Failed to initialize the execution of a plugin in a child process
```

Comment:
reuse errors from pluginprocess?

Reply:
Yes i think that error can be reused to avoid introducing more as this plugin and pluginprocess are quite closely related